docs/labwc-config.5.scd

```diff
@@ -62,19 +62,22 @@ as a comment. Trailing comments are not supported.
 Please note that as labwc reloads the environment file(s) on reconfigure,
 recursive/circular assignments (for example FOO=$FOO:bar) should not be made.
 
-The *autostart* file is executed as a shell script after labwc has read its
+The *autostart* file, if it exists, is executed after labwc has read its
```

Comment:
`session_run_script()` runs `sh %s`, so I think it's better say "as a shell script"